packages/connectivity/src/scp-cf/jwt/jwt.ts

```diff
@@ -74,7 +81,7 @@ export function getSubdomain(
   const decodedJwt = jwt ? decodeJwt(jwt) : {};
   return (
     decodedJwt?.ext_attr?.zdn ||
-    (isXsuaaToken(decodedJwt) ? getIssuerSubdomain(decodedJwt) : undefined)
+    (isNotIasToken(decodedJwt) ? getIssuerSubdomain(decodedJwt) : undefined)
```

Comment:
[req] Can you update the description of this function as well? We consider IAS token and are also no longer strict with XSUAA